src/main/java/com/rcv/ElectionConfig.java

```diff
@@ -405,7 +490,7 @@ private void processCandidateData() {
       }
 
       String uwiLabel = getUndeclaredWriteInLabel();
-      if (uwiLabel != null) {
+      if (uwiLabel != null && !uwiLabel.isEmpty()) {
```

Comment:
Should we actually be excluding `uwiLabel` from `candidatePermutation`? Doesn't that give an unfair disadvantage to UWIs?

Reply:
If uwiLabel exists, it automatically gets eliminated in the first round.